ivy/functional/backends/jax/__init__.py

```diff
@@ -189,3 +190,14 @@ def closest_valid_dtype(type=None, /, as_native=False):
 from .experimental import *
 from . import control_flow_ops
 from .control_flow_ops import *
+
+
+# sub-backends
```

Comment:
I think we should follow the general design with the backends, where we don't import the frameworks even if they're installed unless that backend is set, similarly, we shouldn't import the sub-backend frameworks even if they're installed, unless that sub-backend is set 🙂